app/services/expression_evaluators/measure_unit.rb

```diff
@@ -36,7 +36,7 @@ def measure_unit_answers
     end
 
     def measure_applicable_units
-      commodity.applicable_measure_units.select do |_unit, values|
+      filtered_commodity.applicable_measure_units.select do |_unit, values|
```

Comment:
This reduces the size of the payload returned by the backend significantly for this request because we are only returning the commodity with the measures that apply to our origin geographical area.

Reply:
Behaviourally it's exactly the same